scripts/bootstrap-prefix.sh

```diff
@@ -953,8 +953,9 @@ bootstrap_gnu() {
 	# contain cruft, bug #650284
```

Comment:
Commit message nit: Andreas Schwab is not the Bash maintainer. Just say you're quoting him from the bash-bug ML and give a link.